homework_1/task_02/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"log"
+	"math"
+
+	"code-cadets-2021/homework_1/task_02/taxes"
+)
+
+func getIncome()(float64, error){
```

Comment:
Formatting! In other files as well!

If you're using GoLand `Code` -> `Reformat Code` should do the reformatting automatically for you. 
Alternatively, look into how to run `go fmt` manually.